src/main/java/eggventory/logic/commands/add/AddLoanCommand.java

```diff
@@ -88,4 +90,10 @@ public String execute(StockList list, Ui ui, Storage storage) {
 
         return output;
     }
+
+    // Used by load function
+    public void executeLoadLoanList(LoanList savedLoanList) {
```

Comment:
don't need cos the LoanList isn't accessed as an instance

Reply:
I don't get what you mean. But anyways, this command is needed because what im doing is to create an instance of LoanList in storage to which I am adding the different loans. In Eggventory.java there is a storage.loadTemplateList() function which is a different from storage.load() used for stockist and stocktype.

Reply:
i know what you mean .. ah i don't know if we're even supposed to have an instance of Loanlist